packages/ui/src/components/molecules/Dropdown/stories/Dropdown.stories.tsx

```diff
@@ -64,7 +64,7 @@ const DropdownSelectableTemplate: Story<
     <>
       <Dropdown>
         <DropdownButton>
-          {value} <Caret />
+        <CaretDown /> {value} 
```

Comment:
Should we includes this icon as default to `DropdownButton` component 🤔 ?

Reply:
IDK, because the user could change the positions, sizes, etc. I think better we don't decide this.